src/components.d.ts

```diff
@@ -729,7 +771,7 @@ export namespace Components {
     }
     interface BiggiveImage {
         /**
-          * Image width
+          * Image height
```

Comment:
Good fix, but also illustrative of why IMHO it's better not to add docblocks in the first places unless they say something the code can't say.